elastic/kibana
# [ResponseOps] The count of consecutive active alerts should be available on the alert

Resolves https://github.com/elastic/kibana/issues/175998

## Summary
Follow on work from the alert creation delay feature. This PR adds consecutive_matches, which is the count of active alerts that is used to determine the alert delay, to the aad doc and to the action variables.


### Checklist



### To verify

- Create a new rule with an alert delay
- Add the new `alert.consecutiveMatches` action variable to the action message. Verify that when the alert fires the action variable is populated in the message.
- To verify that the alert docs are as expected, go to [Dev Tools](http://localhost:5601/app/dev_tools#/console) and run the following `GET .internal.alerts-*/_search`
- Go back to the rule alerts table, and add the `kibana.alert.consecutive_matches` field to the table. Verify that it is populated and looks as expected.
